test/LoRaDumpRegisters/LoRaDumpRegisters.ino

```diff
@@ -0,0 +1,26 @@
+/*
+  LoRa register dump
```

Comment:
Is this a duplicate of https://github.com/sandeepmistry/arduino-LoRa/pull/17/files#diff-3d27af960cba75a3af0e3fc42ca29bb2?